besu/src/main/java/org/hyperledger/besu/cli/subcommands/storage/TrieLogSubCommand.java

```diff
@@ -123,6 +131,78 @@ public void run() {
     }
   }
 
+  @Command(
+      name = "export",
+      description =
+          "This command prunes all trie log layers below the retention threshold, including orphaned trie logs.",
+      mixinStandardHelpOptions = true,
+      versionProvider = VersionProvider.class)
+  static class ExportTrieLog implements Runnable {
+
+    @SuppressWarnings("unused")
+    @ParentCommand
+    private TrieLogSubCommand parentCommand;
+
+    @SuppressWarnings("unused")
+    @CommandLine.Spec
+    private CommandLine.Model.CommandSpec spec; // Picocli injects reference to command spec
+
+    @CommandLine.Option(
+        names = "--trie-log-hash",
+        paramLabel = DefaultCommandValues.MANDATORY_LONG_FORMAT_HELP,
+        description = "The hash of the block you want to export the trie log.",
+        arity = "1..1")
+    private String trieLogHash;
+
+    @Override
+    public void run() {
+      TrieLogContext context = getTrieLogContext();
+      final Path dataDirectoryPath =
+          Paths.get(
+              TrieLogSubCommand.parentCommand.parentCommand.dataDir().toAbsolutePath().toString());
+      try {
+        TrieLogHelper.exportTrieLog(
+            context.rootWorldStateStorage(), dataDirectoryPath, Hash.fromHexString(trieLogHash));
+      } catch (IOException e) {
+        throw new RuntimeException(e);
+      }
+    }
+  }
+
+  @Command(
+      name = "import",
+      description =
+          "This command prunes all trie log layers below the retention threshold, including orphaned trie logs.",
+      mixinStandardHelpOptions = true,
+      versionProvider = VersionProvider.class)
+  static class ImportTrieLog implements Runnable {
+
+    @SuppressWarnings("unused")
+    @ParentCommand
+    private TrieLogSubCommand parentCommand;
+
+    @SuppressWarnings("unused")
+    @CommandLine.Spec
+    private CommandLine.Model.CommandSpec spec; // Picocli injects reference to command spec
+
+    @CommandLine.Option(
+        names = "--trie-log-hash",
```

Comment:
Can the exported file contain more than one trielog? Would of expected the import to specify the filename and just import all trielogs in that file